### ISSUE TITLE
"position" vs "location"

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/center-and-variability#482]
For code review #477 ...

> - [ ] PhET prefers to use the term "position" to refer to the physical (x,y) position of objects. This applies to both brands, but is more important for the PhET-iO API. See https://github.com/phetsims/phet-info/issues/126

2 occurrences of "location" in center-and-variability, 15 in soccer-common.   I'm not sure which occurrences are appropriate to change.

**Comment by samreid** [comment]
We will change `getStackAtLocation` to `getStackAtDistance` or `getStackForKickDistance` or something. We'll consult with @catherinecarter to decide the terminology since some of it appears in the PhET-iO API. Then @matthew-blackman will take the lead.

**Comment by catherinecarter** [comment]
In math, position and location are sort of the same thing. 

On a number line, location seems to more often to describe the numerical value at that place. Position seems to be used more in context of points relative to each other rather than an absolute place on the number line, although some textbooks use the language of the location of a point on a number line is the distance it is from zero.

Bottom line is that there doesn't seem to be a 'standard' for using the words position or location on a number line, or at least there doesn't seem to be an argument over one being more correct than the other. 

So whatever PhET is/has adopted as the convention seems to be fine in this context. If there are specific names in the code where there's a question, I'd be happy to take a look.

**Comment by samreid** [comment]
From today's discussion, we will use `getStackAtValue` etc.

**Comment by samreid** [comment]
@marlitas and I reviewed it, and the changes look great. We added a few more rewordings and are ready to close.
